### PR TITLE
support web

### PR DESCRIPTION
**Pull request by melodysdreamj**
### Thanks a lot for contributing!<br>
Provide a description of your changes below

The dart:io package is not supported on the web, and because of this, it is marked as not supported for web on pub.dev. Therefore, dart:io is conditionally used only in non-web environments, and in web environments, it is replaced with suitable alternative code.

### CODE REVIEW

File: packages/talker_logger/lib/src/logger_io.dart [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  TalkerLogger({
+    TalkerLoggerSettings? settings,
+    this.formatter = const ExtendedLoggerFormatter(),
+    LoggerFilter? filter,
+    void Function(String message)? output,
+  }) {
+    this.settings = settings ?? TalkerLoggerSettings();
+    // ignore: avoid_print
+    _output = output ??
+        (String message) => message.split('\n').forEach(stdout.writeln);
```
Comment (Frezyx) [inline packages/talker_logger/lib/src/logger_io.dart:15]: The only difference between logger_io and logger_web is in this line
Please make outout_io.dart and output_web.dart files with **output(String message)** method realizations 

We can't make two almost identical files

**Review by Frezyx — CHANGES_REQUESTED** [review]
Hello @melodysdreamj ! 
Thanks a lot for this contribution, but please fix review issues

File: packages/talker_logger/lib/src/logger_io.dart (hunk 1 above)
Comment (melodysdreamj) [reply packages/talker_logger/lib/src/logger_io.dart]: Hello @Frezyx 

As you mentioned, I've added only the changes to the two files and preserved most of the original code. The modified code runs well on both web and Android, but on Android, stdout.writeln did not work, so I changed it to print. Thank you.

File: packages/talker_logger/lib/src/logger_io.dart [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import 'dart:io';
+
+void outputLog(String message) => message.split('\n').forEach(print);
```
Comment (Frezyx) [inline packages/talker_logger/lib/src/logger_io.dart:3]: Please use **stdout.writeln** here 
**print** method used in **TalkerFlutter.init()** method from talker_flutter lib

https://github.com/Frezyx/talker/blob/master/packages/talker_flutter/lib/src/extensions/talker_flutter.dart

**Review by Frezyx — CHANGES_REQUESTED** [review]

File: packages/talker_logger/lib/src/logger_io.dart (hunk 2 above)
Comment (melodysdreamj) [reply packages/talker_logger/lib/src/logger_io.dart]: Thank you for your kind guidance. As you suggested, I've changed "print" to "stdout.writeln". Thank you.

**Review by Frezyx — APPROVED** [review]

**Comment by Frezyx** [comment]
@melodysdreamj Thank you for this great contribution!  ❤️ 
I will make new release version soon

**Comment by melodysdreamj** [comment]
Thank you for your kind help 🙏

**Pull request merged**